src/Cms/Users.php

```diff
@@ -21,12 +22,29 @@
  * @license   https://getkirby.com/license
  *
  * @template TUser of \Kirby\Cms\User
- * @extends \Kirby\Cms\Collection<TUser>
+ * @extends \Kirby\Cms\LazyCollection<TUser>
  */
-class Users extends Collection
+class Users extends LazyCollection
 {
 	use HasUuids;
 
+	/**
+	 * Creates a new Collection with the given objects
+	 *
+	 * @param iterable<TValue> $objects
```

Comment:
```suggestion
	 * @param iterable<TUser> $objects
```
I think this needs to be `TUser`